hypervisor/src/vm.rs

```diff
@@ -359,6 +370,26 @@ pub trait Vm: Send + Sync + Any {
     }
     /// Downcast to the underlying hypervisor VM type
     fn as_any(&self) -> &dyn Any;
+    /// Import the isolated pages
+    #[cfg(feature = "sev_snp")]
+    fn import_isolated_pages(
+        &self,
+        _page_type: u32,
+        _page_size: u32,
+        _pages: &[u64],
+    ) -> Result<()> {
+        unimplemented!()
```

Comment:
Can't we already just call the implementation in `mshv/mod.rs`?

Reply:
This is the default implementation for KVM. Otherwise, I guess we have to add another mshv feature gate here